index.bs

```diff
@@ -10141,87 +10093,67 @@ which allows construction of objects that
 implement the interface on which the
 [{{NamedConstructor}}] extended attributes appear.
```

Comment:
I would remove the "must have a [[Call]] property" sentence; that is implicit and restating it is confusing (how could it not be true)?